.travis.yml

```diff
@@ -1,13 +1,23 @@
 sudo: false
 language: scala
+services:
+- redis-server
 scala:
 - 2.11
 jdk:
 - oraclejdk8
 env:
   global:
-  - BRIDGE_USER=travis
+  - BRIDGE_USER=heroku
```

Comment:
Don't do this. The travis user is travis. heroku user is used for the actual Bridge Server. Having Travis and Bridge use the same user will cause conflicts.